### DISCUSSION TITLE
How do i change the max filesize?

### DISCUSSION THREAD

**Discussion by aradox** [pocketbase/pocketbase#829] [Q&A] [answered]
Were can i turn off the max file size?
![filesize](https://user-images.githubusercontent.com/4798993/197417137-fd62dcf9-3ad7-4562-b805-1a313b857be1.JPG)

I need to be able to upload files to the max of 1GB but the max is 40MB now, where do i change this?

**Comment by ganigeorgiev** [comment, marked as answer]
You can change the file size limit and other file validators from the collection field options:

![example_screenshot](https://i.imgur.com/onCbSDF.png)

The default is 5mb (note that it is in bytes!).

Also keep in mind that I haven't tested really large file uploads and by default the multipart/form-data body will be kept in the memory for up to 32mb; anything above this will be stored on the disk into temporary files.
